packages/devtools_app/lib/src/shared/common_widgets.dart

```diff
@@ -646,18 +651,18 @@ class DevToolsTooltip extends StatelessWidget {
   })  : assert((message == null) != (richMessage == null)),
         super(key: key);
 
-  final String message;
-  final InlineSpan richMessage;
+  final String? message;
+  final InlineSpan? richMessage;
   final Widget child;
   final Duration waitDuration;
   final bool preferBelow;
-  final EdgeInsetsGeometry padding;
-  final Decoration decoration;
-  final TextStyle textStyle;
+  final EdgeInsetsGeometry? padding;
```

Comment:
this can be non-nullable since we pass a default value

Reply:
we pass nullable value here in maybeWrapWithTooltip

Reply:
ack